buzzmobile/process/frame_merger/frame_merger.py

```diff
@@ -15,7 +14,7 @@ def __init__(self):
 
 # Global Variables
 bridge = CvBridge()
-world_pub = rospy.Publisher('world_model', Image, queue_size=0)
+world_pub = rospy.Publisher('world_model', Image, queue_size=1)
```

Comment:
For future reference, having `queue_size=0` make ros use an unlimited sized queue, which just explodes your computer's RAM :)